newTab.html

```diff
@@ -4,7 +4,7 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
-<html>
+<html manifest="newtab.appcache">
```

Comment:
I agree. Since we're using Service Workers, and if the cache is ready, let's do this.